resource/storage.go

```diff
@@ -155,7 +155,7 @@ func (s storageWrapper) MultiGet(ctx context.Context, ids []interface{}) (items
 }
 
 // Find tries to use storer MultiGet with some pattern or Find otherwise
-func (s storageWrapper) Find(ctx context.Context, lookup *Lookup, page, perPage int) (list *ItemList, err error) {
+func (s storageWrapper) Find(ctx context.Context, lookup *Lookup, page, perPage, offset int) (list *ItemList, err error) {
```

Comment:
I think I understand what you mean. the math is the same. but a client requesting with `page` has a different intention behind it as one who wants an offset.

so, while:
```
Find(ctx, query, 2, 5, 0) // sets page to 2
Find(ctx, query, 0, 5, 5*(2-1)) // sets offset instead of page
```

may be the same. this isn't:
```
curl http://localhost:8080/users?offset=10&limit=10
curl http://localhost:8080/users?page=2&limit=10
```